### PR TITLE
Fix code formatting

### PR DESCRIPTION
**Pull request by jperedadnr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,12 +104,15 @@ Using brew:
 
 * Build the native image:
 
-    mvn clean client:build
+```
+mvn clean client:build
```
Comment (Alain-Bearez) [inline README.md]: Missing closing triple back tick.

File: README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ For example:
 
 Using brew:
 
-    `brew install --HEAD libusbmuxd
+    brew install --HEAD libusbmuxd
```
Comment (Alain-Bearez) [inline README.md]: In order to keep consistency, I would have enclosed this line into triple ticks, too.

File: README.md (hunk 1 above)
Comment (jperedadnr) [reply README.md]: Good catch, fixed

File: README.md (hunk 2 above)
Comment (jperedadnr) [reply README.md]: Makes sense, but will require changing all the above code blocks as well. For now it can be as is.

**Pull request merged**